src/cases/components/task-alert-banner/task-alert-banner.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'exui-task-alert-banner',
+  templateUrl: './task-alert-banner.component.html',
+  styleUrls: ['./task-alert-banner.component.scss']
+})
+export class TaskAlertBannerComponent {
+  public alertTitle = 'Task alert';
+  public alertMessage = 'An application is pending a decision. Consider if this application has an impact on any other tasks you might wish to complete.';
```

Comment:
```suggestion
  @Input() public alertTitle;
  @Input() public alertMessage;
```
If this is component, it should be a generic component and passed from the container.

Reply:
Yes, that is a good point. Will change